### ISSUE TITLE
Stuttering in Proton

### ISSUE THREAD

**Issue by NikoBellicRU** [ValveSoftware/Proton#3853]
Why Proton has so much stutter? I mean i know that its the shaders compiling (i think) but wasn't "fossilize" supposed to fixed this issue? Also why Proton-GE has significant reduced stutter (some games no stutter at all) than proton? Some games its night and day difference so i'm not sure if its Async or something else but it's really good, couldn't whatever this is be implemented in official proton?

UPDATE : 
Resident Evil 5 - https://www.youtube.com/watch?v=5P39jSesmSs
Batman Arkham City - https://www.youtube.com/watch?v=1Ei2Dcif0eU&feature=youtu.be

**Comment by NTMan** [comment]
> Also why Proton-GE has significant reduced stutter

Maybe because Proton-GE has DXVK with async patches?

**Comment by NikoBellicRU** [comment]
> > Also why Proton-GE has significant reduced stutter
> 
> Maybe because Proton-GE has DXVK with async patches?

Yeah now i know for sure that it's the new Async patch that he has on this new version (5.8), I posted this on Linux gaming reddit.

https://www.reddit.com/r/linux_gaming/comments/ghhhto/protonge_reduced_a_lot_of_my_stuttering/?utm_medium=android_app&utm_source=share

**Comment by Mushoz** [comment]
Async shaders can get you banned in online play. Are you using an AMD GPU by any chance? Just switch from LLVM to ACO and stuttering is mostly fixed anyway. And it won't get you banned.

**Comment by NikoBellicRU** [comment]
> Async shaders can get you banned in online play. Are you using an AMD GPU by any chance? Just switch from LLVM to ACO and stuttering is mostly fixed anyway. And it won't get you banned.

I replied on reddit but I'll post it here also:

I use Nvidia and that's not relevant, like I said this should be enabled to all games but the ones that can have those issues like for example overwatch, there's a lot of single player games that can still use this and even some multiplayer that don't have any sort of anti cheat system,we shouldn't punish all games cause of some that have that problem (just disable Async for them)

Maybe in the future will be a better solution but for now this new patch has been awesome in every game I tried and I just can't unsee/unuse it now that i've experienced it, again it was night and day diference.

**Comment by kisak-valve** [comment]
Hello @NikoBellicRU, that has already been evaluated on the dxvk issue tracker at https://github.com/doitsujin/dxvk/issues/1026#issuecomment-486201661. Async shader compilation will not be supported in Proton.

**Comment by NikoBellicRU** [comment]
> Hello @NikoBellicRU, that has already been evaluated on the dxvk issue tracker at [doitsujin/dxvk#1026 (comment)](https://github.com/doitsujin/dxvk/issues/1026#issuecomment-486201661). Async shader compilation will not be supported in Proton.

That's unfortunate, is it possible to at least provide a command to enable it like GE?

If not I guess fossilize it the way to go but when will it be enabled by default? (I mean fossilize replay)

**Comment by NTMan** [comment]
My 5 cents: I am very sad that async patches are considered unacceptable because I am really feeling smoother gameplay even when the computer is loaded with heavy tasks. ACOs have no such effect. I am a user of the top Linux PC Ryzen 3950X + Radeon VII and play exclusively in 4K with ultra settings.

**Comment by NikoBellicRU** [comment]
> My 5 cents: I am very sad that async patches are considered unacceptable because I am really feeling smoother gameplay even when the computer is loaded with heavy tasks. ACOs have no such effect. I am a user of the top Linux PC Ryzen 3950X + Radeon VII and play exclusively in 4K with ultra settings

Right? I really think they should reconsider using Async or at least part of it I mean, just for fun I been changing from from Async to default (while deleting caches) just to see if it's really that big of a deal and yeah it is, at least until another solution comes out I cannot play without it

**Comment by NikoBellicRU** [comment]
I made a little video to show this a bit better but when i uploaded to youtube the quality got nerfed a lot but i think you can still see it well, if not watch the graph
https://www.youtube.com/watch?v=5P39jSesmSs

**Comment by NikoBellicRU** [comment]
I posted another video in Batman Arkham City
https://www.youtube.com/watch?v=1Ei2Dcif0eU&feature=youtu.be

**Comment by NikoBellicRU** [comment]
So i tried the new beta client with the fossilize replay but it's not working for me, i was wondering if it was still disabled since in the steam log nothing about this is mencioned but on reddit people are saying that it's working with amd and nvidia so i don't get it.

I tried everything at this point, reinstall system, installed latest kernel,latest nvidia vulkan beta driver (440.66.15), deleted the shaders folder and wait for it to compile again (gta 5 took me 5m) and nothing.

In the fozpipelinev4 folder the cache doesn't seem to be in use cause soon as i open the game it just creates a new cache and keeps increasing size the more i play.
